cmd/object-handlers.go

```diff
@@ -665,6 +632,12 @@ func (api objectAPIHandlers) getObjectHandler(ctx context.Context, objectAPI Obj
 
 	setHeadGetRespHeaders(w, r.Form)
 
+	var buf *bytebufferpool.ByteBuffer
+	if update && globalCacheConfig.MatchesSize(objInfo.Size) {
```

Comment:
Line 613-Line 614 remove please and then LGTM
Tested.

Reply:
why? @jiuker

Reply:
We save the encrypted objects in cache. @harshavardhana. Configured  KMS is SEE